### ISSUE TITLE
[bug]: Cannot select rows, relevant buttons are not disabled by default and css-styling doesn't apply in metadata-view

### ISSUE THREAD

**Issue by signekb** [seedcase-project/seedcase-sprout#345]
After #300 was merged into main, `metadata-view` doesn't work as expected.
1. I cannot select rows
2. the edit and upload buttons are not disabled by default, and 
3. the css styling (e.g., hovering) doesn't work

**Comment by signekb** [comment]
This is fixed now.
